compiler/src/dmd/dsymbol.h

```diff
@@ -430,6 +430,7 @@ namespace dmd
 {
     void addMember(Dsymbol *dsym, Scope *sc, ScopeDsymbol *sds);
     Dsymbol *search(Dsymbol *d, const Loc &loc, Identifier *ident, SearchOptFlags flags = (SearchOptFlags)SearchOpt::localsOnly);
+    Dsymbols *include(Dsymbol d, Scope *sc);
```

Comment:
```diff
-Dsymbols *include(Dsymbol d, Scope *sc);
+Dsymbols *include(Dsymbol *d, Scope *sc);
```

Reply:
It failed as well `Error: undefined reference to dmd::include(Dsymbol*, Scope*)  referenced from _ZN15MiniGlueVisitor5visitEP10TypeStruct`

I think it has to do with the arguments here in defintion in dsymbolsem.d 
`extern(C++) Dsymbols* include(Dsymbol d, Scope* sc)`

Reply:
ohh, It doesn't. looking at it right now